editor-components/stats/stats2/stats2.module.scss

```diff
@@ -1,114 +1,265 @@
+@import "../../../composer-base-components/base/utitilities/viewports.module.scss";
+
 .container {
-    width: 100%;
-    padding: 50px 0;
-    .max-content {
-        max-width: var(--composer-content-width);
-        margin: auto;
-        .bottom-child {
+  color: var(--composer-font-color-primary);
+  display: flex;
+
+  .max-content {
+    display: flex;
+    flex-direction: column;
+
+    .header-wrapper {
+      width: 60%;
+
+      &.full-width {
+        min-width: 100%;
+      }
+
+      .header {
+        word-break: break-word;
+      }
+    }
+
+    .bottom-content {
+      flex-wrap: unset;
+
+      .subHeader {
+        flex: 1;
+        position: sticky;
+        top: 5rem;
```

Comment:
when the font size changed the top position also will changed. is that expected?